Thrive.csproj

```diff
@@ -466,6 +477,8 @@
     <Compile Include="src\modding\NewModGUI.cs" />
     <Compile Include="src\modding\WorkshopData.cs" />
     <Compile Include="src\saving\FileHelpers.cs" />
+    <Compile Include="src\saving\FossilisedSpecies.cs" />
+    <Compile Include="src\saving\FossilisedSpeciesInformation.cs" />
```

Comment:
Specific saveable types shouldn't be in the saving folder but once again in the thriveopedia subfolder I suggested above.